cdap-api/src/main/java/co/cask/cdap/api/workflow/NodeValueEntry.java

```diff
@@ -18,14 +18,14 @@
 
 /**
  * Multiple nodes in the Workflow can add the same key to the {@link WorkflowToken}.
- * This class provides a mapping from node name to the value which was set for the
+ * This class provides a mapping from node name to the {@link Value} which was set for the
  * specific key.
  */
 public final class NodeValueEntry {
```

Comment:
Yes. It can be. However not sure what will be the benefit. Maybe `Entry` in the name of the class is confusing, since this is not the key-value entry but only represents the value part. Perhaps rename the class to `NodeValue` and keep it outside of the `WorkflowToken`?